docs/docs/components/lifecycle-operator/webhook.md

```diff
@@ -0,0 +1,68 @@
+---
+comments: true
+---
+
+# Admission Webhooks
+
+Keptn uses
+[Admission Webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks)
+to mutate resources.
+To enable the webhook (and therefore Keptn Lifecycle Management)
+for a certain namespace, the namespace must be annotated:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: podtato-kubectl
+  annotations:
+    keptn.sh/lifecycle-toolkit: "enabled"  # this line tells the webhook to handle the namespace
+```
+
+The mutating webhook only modifies specifically annotated resources in the annotated namespace.
+When the webhook receives a request for a new pod,
+it looks for the workload annotations:
+
+```yaml
+keptn.sh/workload: "some-workload-name"
+```
+
```

Comment:
The entire bit from her down is hinting at all the things the webhook does but it kinda misses a clear order, list of them... I got lost in the text and the part about auto gen of the app and version was also not superclear... 

maybe we can add a list of things it does? Reordering the bit after to match more the logic?